app/components/peopletable/peopletable.js

```diff
@@ -86,11 +92,17 @@ const PeopleTable = translate()(class PeopleTable extends React.Component {
       showModalOverlay: false,
       dialog: '',
       tableHeight: 590,
+      showSearchReset: false,
     };
 
     WindowSizeListener.DEBOUNCE_TIME = 50;
   }
 
+  handleEmptySearch(e){
```

Comment:
Since this event callback call setState() followed by handleFilterChange(), which calle a setState() too, with a test on the filter value, it may be safer to write this function:
```js
  handleEmptySearch(e) {
    this.handleFilterChange(null);
  }
```